tests/mock/dist.rs

```diff
@@ -153,6 +155,11 @@ impl MockDistServer {
             } else {
                 None
             };
+            let zst_hash = if enable_zst {
```

Comment:
This to me looks like a loop trying to break free. Something for the future I guess.

Reply:
Yeah, it'll be nice to make this more generic and loopy in the future.